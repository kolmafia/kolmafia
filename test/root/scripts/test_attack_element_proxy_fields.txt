ash $monster[warwelf].attack_element
ash $monster[warwelf].attack_elements
ash $monster[Knott Yeti].attack_element
ash $monster[Knott Yeti].attack_elements
ash $monster[lihc].attack_element
ash $monster[lihc].attack_elements
