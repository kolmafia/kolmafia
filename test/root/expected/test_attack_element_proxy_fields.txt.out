Returned: none
image => circle.gif
Returned: aggregate boolean [element]
none => true
Returned: cold
image => snowflake.gif
Returned: aggregate boolean [element]
cold => true
Returned: spooky
image => skull.gif
Returned: aggregate boolean [element]
sleaze => true
spooky => true
